test/gurka/test_route.cc

```diff
@@ -247,7 +247,7 @@ TEST(AutoDataFix, deprecation) {
 class AlgorithmTest : public ::testing::Test {
 protected:
   static gurka::map map;
-  static uint32_t current, historical, constrained;
+  static uint32_t current, historical, constrained, freeflow;
```

Comment:
i used this test as a model for what i did in the map matching test. they are basically idential and return identical results